app/lib/sidebar/local_widgets/list_tile.dart

```diff
@@ -1,47 +1,55 @@
 import 'package:app/navbar/navigation.dart';
+import 'package:app/style/style.dart';
 import 'package:flutter/material.dart';
 
-class CustomTile extends StatelessWidget {
+class TemplateListTile extends StatelessWidget {
   final IconData icon;
   final String text;
+  final TextStyle textStyle;
 
-  CustomTile({@required this.icon, @required this.text}) {
+  TemplateListTile(
+      {@required this.icon, @required this.text, @required this.textStyle}) {
     assert(icon != null);
     assert(text != null);
+    assert(textStyle != null);
   }
 
   @override
   Widget build(BuildContext context) {
-    return Padding(
-      padding: const EdgeInsets.fromLTRB(8.0, 0, 8.0, 0),
-      child: Container(
-        decoration: BoxDecoration(
-          border: Border(
-            bottom: BorderSide(color: Colors.grey[400]),
-          ),
+    return Container(
+      decoration: BoxDecoration(
+        border: Border(
+          bottom: BottomBorderStyle,
         ),
-        child: InkWell(
-          onTap: () {
-            // TODO: Implement a better way of routing
-            Navigator.pop(context);
-            Navigator.push(
-              context,
-              MaterialPageRoute(
-                  builder: (context) => NavigationBarController()),
-            );
-          },
-          child: Row(
-            children: [
-              Padding(
-                padding: const EdgeInsets.all(5.0),
-                child: Icon(icon),
-              ),
-              Padding(
-                padding: const EdgeInsets.all(8.0),
-                child: Text(text),
+      ),
+      child: InkWell(
+        onTap: () {
+          // TODO: Implement a better way of routing
+          Navigator.pop(context);
+          Navigator.push(
+            context,
+            MaterialPageRoute(builder: (context) => NavigationBarController()),
+          );
+        },
+        child: Row(
+          children: [
+            Padding(
+              /*
+                The padding property can be added to style.dart,
+                but doing so would be an overkill.
+                Source: https://stackoverflow.com/questions/44053363/flutter-padding-for-all-widgets
```

Comment:
Seems fair! Might wanna move this comment to `styling.dart` to prevent someone from adding it in the future, and the developer will see it when trying to find a common padding style.

- [ ] Move comment to `styling.dart`